core/src/components/modal/modal.tsx

```diff
@@ -146,6 +146,9 @@ export class Modal implements ComponentInterface, OverlayInterface {
    * The interaction behavior for the sheet modal when the handle is pressed.
    * Defaults to `"none"`, which  means the modal will not change size or position when the handle is pressed.
    * Set to `"cycle"` to let the modal cycle between available breakpoints when pressed.
+   *
+   * Handle behavior is only available when the `handle` property is set to `true` and
```

Comment:
The `handle` property defaults to `undefined` and is enabled by default, so "Handle behavior is only available when the `handle` property is set to `true`" is not accurate.